### PR TITLE
Setup TGIS for RHODS 1.32

### PR DESCRIPTION
**Pull request by heyselbi**
- Setting up TGIS for RHODS 1.32
- Updated nightly to final tag
- added the files from running `make ci-operator-config jobs prow-config`

### CODE REVIEW

File: ci-operator/config/red-hat-data-services/text-generation-inference/red-hat-data-services-text-generation-inference-rhods-1.32.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  '*':
+    limits:
+      cpu: "20"
+      memory: 16Gi
+    requests:
+      cpu: "20"
+      memory: 16Gi
+tests:
+- as: nightly-image-mirror
+  cron: 0 0 1 * * ?
```
Comment (Jooho) [inline ci-operator/config/red-hat-data-services/text-generation-inference/red-hat-data-services-text-generation-inference-rhods-1.32.yaml:28]: please use postsubmit. specific branch does not need to be built every night

**Comment by heyselbi** [comment]
/pj-rehearse pull-ci-red-hat-data-services-text-generation-inference-rhods-1.32-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-red-hat-data-services-text-generation-inference-rhods-1.32-images

**Comment by heyselbi** [comment]
/pj-rehearse pull-ci-red-hat-data-services-text-generation-inference-rhods-1.32-images

**Pull request merged**